src/app/Home.module.css

```diff
@@ -1,33 +1,14 @@
-.landingBackground h1 {
-    font-size: 2em;
-    margin-bottom: 1rem;
-    margin-left: 1em;
-    margin-right: 1em;
-}
-  
-.landingBackground h2 {
-    font-size: 1.5em;
-    margin-bottom: 0.5rem;
-    margin-left: 1em;
-    margin-right: 1em;
-}
-
-.landingBackground p {
-    margin-bottom: 0.5rem;
-    margin-left: 1em;
-    margin-right: 1em;
-}
-  
-.centerText {
+.content {

```

Comment:
The `.content` class is currently the same for each page, but I'm opting not to condense these as that may not be true for long. If we find this is common for other pages too we can abstract it then (WET - Write Everything Twice).